core/src/main/java/io/questdb/cairo/AbstractFullPartitionFrameCursor.java

```diff
@@ -27,18 +27,21 @@
 import io.questdb.cairo.sql.PartitionFrame;
 import io.questdb.cairo.sql.PartitionFrameCursor;
 import io.questdb.cairo.sql.StaticSymbolTable;
+import io.questdb.griffin.engine.table.parquet.PartitionDecoder;
 import io.questdb.std.Misc;
 import org.jetbrains.annotations.TestOnly;
 
 public abstract class AbstractFullPartitionFrameCursor implements PartitionFrameCursor {
     protected final FullTablePartitionFrame frame = new FullTablePartitionFrame();
+    protected final PartitionDecoder parquetDecoder = new PartitionDecoder();
```

Comment:
can we lazy initialize this?

Reply:
Done in 99216c6373c417ce1d29d94778a1cba08e6f921d